src/js/components/DateTimeFormatPicker.react.js

```diff
@@ -11,13 +11,14 @@
 const React = require('react');
 const classNames = require('classnames');
 const moment = require('moment');
+const phpDateTimeMomentFormat = require('../utils/phpDateTimeMomentFormat');
```

Comment:
I am getting an unused var warning here. Is there a way to avoid it? I understand that you are adding a function, so you need to load the file.